src/redux/consigureStore.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from '@reduxjs/toolkit';
+import bookReducer from './book/book';
+import categories from './categories/categories';
+
+const allReducers = combineReducers({
+  books: bookReducer,
+  categories,
+});
+
+export default allReducers;
```

Comment:
- [ ] it would be better to use `ConfigureStore` as required in the requirements as configureStore is the new way to create the combine the reducers 
- Screenshot of the requirements 
<img width="733" alt="Screenshot 2022-11-14 at 15 39 59" src="https://user-images.githubusercontent.com/61790587/201688086-91fe6a9d-14a6-449c-af6b-27a8d7ac2fa8.png">